gramex/handlers/mlhandler.py

```diff
@@ -16,11 +16,20 @@
 import pandas as pd
 from sklearn.compose import ColumnTransformer
 from sklearn.pipeline import Pipeline
-from sklearn.preprocessing import OneHotEncoder, StandardScaler
+from sklearn.metrics import roc_auc_score
+from sklearn.preprocessing import OneHotEncoder, StandardScaler, LabelEncoder
 from slugify import slugify
 from tornado.gen import coroutine
 from tornado.web import HTTPError
 from sklearn.metrics import get_scorer
+try:
+    from transformers import pipeline
```

Comment:
Let's have a consistent policy for imports. Treat torch and huggingface the same way.

Reply:
@sanand0  Fixed it.